.travis.yml

```diff
@@ -2,7 +2,6 @@ language: node_js
 sudo: false
 node_js:
   - 4
-  - 6
```

Comment:
are we dropping support for node 6 because we'll be splitting out the functionality?